code/game/objects/structures/hybrisa_props.dm

```diff
@@ -0,0 +1,2359 @@
+// Hybrisa props
+
+/obj/structure/prop/hybrisa
+	icon = 'icons/obj/structures/prop/urban/vehiclesexpanded.dmi'
+	icon_state = "pimp"
+
+// Vehicles
+
+/obj/structure/prop/urban/vehicles
+	icon = 'icons/obj/structures/prop/urban/vehiclesexpanded.dmi'
+	icon_state = "SUV"
+	max_integrity = 3000
+	var/damage_state = 0
+	var/brute_multiplier = 1
+	resistance_flags = XENO_DAMAGEABLE
+
+/obj/structure/prop/urban/vehicles/take_damage(damage_amount, damage_type = BRUTE, armor_type = null, effects = TRUE, attack_dir, armour_penetration = 0, mob/living/blame_mob)
+	. = ..()
+	if(!.)
+		return
+	switch(obj_integrity)
+		if(2500 to 3000)
+			icon_state = initial(icon_state)
+			return
+		if(2000 to 2500)
+			icon_state = icon_state = "[initial(icon_state)]_damage_[1]"
+		if(1500 to 2000)
+			icon_state = icon_state = "[initial(icon_state)]_damage_[2]"
+		if(1000 to 1500)
+			icon_state = icon_state = "[initial(icon_state)]_damage_[3]"
+		if(500 to 1000)
+			icon_state = icon_state = "[initial(icon_state)]_damage_[4]"
+		if(0 to 500)
+			icon_state = icon_state = "[initial(icon_state)]_damage_[5]"
+
+/obj/structure/prop/urban/vehicles/proc/explode(dam, mob/M)
+	src.visible_message(span_danger("<B>[src] blows apart!</B>"), null, null, 1)
+	playsound(loc, 'sound/effects/car_crush.ogg', 25)
+	var/turf/Tsec = get_turf(src)
+	new /obj/item/stack/rods(Tsec)
+	new /obj/item/stack/rods(Tsec)
+	new /obj/item/stack/sheet/metal(Tsec)
+	new /obj/item/stack/sheet/metal(Tsec)
+	new /obj/item/stack/cable_coil/cut(Tsec)
+
+	new /obj/effect/spawner/gibspawner/robot(Tsec)
+	new /obj/effect/decal/cleanable/blood/oil(src.loc)
+
+	deconstruct(FALSE)
+
+/obj/structure/prop/urban/vehicles/suv
+	icon = 'icons/obj/structures/prop/urban/vehiclesexpanded.dmi'
+	icon_state = "SUV"
+
+/obj/structure/prop/urban/vehicles/suvdamaged
+	icon = 'icons/obj/structures/prop/urban/vehiclesexpanded.dmi'
+	icon_state = "SUV_damaged"
```

Comment:
these are defined later on in this file as well? Not sure if its just these two, I suspect not.

Reply:
these still need fixing